### PR TITLE
Configures app for Heroku

### PR DESCRIPTION
**Pull request by amoose**
#88 included the Procfile which adds background worker role to Heroku

Adds script to setup and launch Heroku instance
Updates secrets.yml
Updates README with instructions

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -24,5 +24,4 @@ test:
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
-  db_key_base: 92ffd31f3cb355b7252afc4633d60d10e6e2266a5e0b8e456ddf10d3519e4cda2303dbe136d1ae42c168e4ce17ae327c21b0f473cfed22928e4f46700db4d828
```
Comment (amoose) [inline config/secrets.yml]: Unused key, correct?

Comment (jgrevich) [inline config/secrets.yml]: Yep, that was leftover from using GitlabHQ

**Comment by jgrevich** [comment]
Awesome!!! My only suggestion is that we update the README to reflect that this doesn't yet include the work needed to setup and configure background workers in Heroku (unless I'm wrong about that).

Please rebase and we'll get this merged in for the upcoming release.

**Comment by amoose** [comment]
We are actually already supporting bg workers on Heroku and CloudFoundry! thanks to @jmcarp and #85

**Comment by amoose** [comment]
Rebased! 📯

**Comment by jgrevich** [comment]
:cloud_with_lightning: Works great. 

That's huge, thanks. I love the heroku 1-click deploy:

##### Use this friendly button:
[Deploy](https://heroku.com/deploy?template=https://github.com/heroku/node-js-sample)

**Pull request merged**